examples/background.rb

```diff
@@ -0,0 +1,39 @@
+require 'optparse'
+$LOAD_PATH.unshift("../lib") # manageiq-messaging
```

Comment:
Prefer bundle/setup over manipulating the LOAD_PATH (bundle/setup will do that automatically and as a bonus you don't need to call bundle exec on the CLI)